hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncProcess.java

```diff
@@ -153,12 +153,13 @@ public void waitUntilDone() throws InterruptedIOException {
   private final int periodToLog;
 
   AsyncProcess(ClusterConnection hc, Configuration conf, RpcRetryingCallerFactory rpcCaller,
-    RpcControllerFactory rpcFactory) {
-    this(hc, conf, rpcCaller, rpcFactory, hc.getConnectionConfiguration().getRetriesNumber());
+    RpcControllerFactory rpcFactory, Map<String, byte[]> requestAttributes) {
```

Comment:
The changes to AsyncProcess are a little ugly imo, but this was the most straightforward way to get request attribute support in HTable